project.mml

```diff
@@ -2180,7 +2180,7 @@ Layer:
             tags->'addr:unit' AS addr_unit,
             way_area/NULLIF(!pixel_width!::real*!pixel_height!::real,0) AS way_pixels
           FROM planet_osm_polygon
-          WHERE (("addr:housenumber" IS NOT NULL) OR ("addr:housename" IS NOT NULL) OR (tags->'addr:unit' IS NOT NULL))
+          WHERE (("addr:housenumber" IS NOT NULL) OR ("addr:housename" IS NOT NULL) OR ((tags->'addr:unit') IS NOT NULL))
```

Comment:
This should be `OR NOT tags ? 'addr:unit'`

Reply:
As I do not fully understand the syntax, I would prefer if someone else could do this change.